en_US/data-integration/snowflake.md

```diff
@@ -39,12 +39,33 @@ This section introduces the preparations required before creating a Snowflake Si
 
 ### Prerequisites
 
-- Understanding of [rules](./rules.md).
-- Understanding of [data integration](./data-bridges.md).
+- Understanding of EMQX [rules](./rules.md) and [data integration](./data-bridges.md) concepts.
+- A working Snowflake account with admin privileges.
+
+### Choose Upload Mode
+
+::: tip
+
+Choose the mode first, as it determines how you configure both EMQX and your Snowflake environment.
+
+:::
+
+EMQX supports two modes for sending data to Snowflake:
+
+| Mode       | Description                                                  | Requires ODBC |
+| ---------- | ------------------------------------------------------------ | ------------- |
+| Aggregated | EMQX buffers MQTT messages into local files, then uploads them to a Snowflake stage. A pipe, configured with a `COPY INTO` statement, automatically loads those staged files into a target table. For more details, see [Snowflake Snowpipe Documentation](https://docs.snowflake.com/en/user-guide/data-load-snowpipe-intro). | Yes           |
+| Streaming  | Sends data in real time via the Snowpipe Streaming API (AWS-only), writing rows directly to Snowflake tables. | No            |
```

Comment:
The way our connector is currently structured, it still requires ODBC installed even though it doesn't use it for this mode...  :sweat: 

We maybe could improve this in the future by attempting to initialize the ODBC connection pool lazily only when the first aggregated action is added.  But, for now, it always starts an ODBC connection.